src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -284,6 +285,95 @@ private void testEditQuestionAction() throws Exception {
         feedbackEditPage.clickAndCancel(feedbackEditPage.getCancelQuestionLink(-1));
     }
     
+    private void testEditVisibilityAction() throws Exception {
+
+        ______TS("edit question 1 to Giver's team members and Giver");
```

Comment:
I have encountered a problem actually. When I try to test that the row "Giver's team member and giver" is displayed or not, it is actually displayed. But when I check the text row by row, it did not show up.